### PR TITLE
Feature/testing 

### PR DESCRIPTION
**Pull request by Tchilo**
- Add __test__ folder
- Add operate.test.js
- Add calculate.test.js
- Add Home.test.js
- Add Quote.test.js
- Add Nav.test.js
- Use Jest snapshots to test the components

### CODE REVIEW

File: src/__tests__/Calculator.test.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    cleanup();
+  });
+
+  test('Calculator page render', () => {
+    render(<Calculator />);
+    const acBtn = screen.getByText('AC');
+    expect(acBtn).toBeInTheDocument();
+  });
+
+  test('Calculator calculations', () => {
```
Comment (afanimartin) [inline src/__tests__/Calculator.test.js:16]: - Please add test scenarios for: `+/-`, `.`, `=` operators. It's mandatory that you test all functionalities in this module.

**Review by afanimartin — CHANGES_REQUESTED** [review]
Hi @Tchilo, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

### Martin Afani
#### Slack: whiz25

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/__tests__/Calculator.test.js (hunk 1 above)
Comment (Tchilo) [reply src/__tests__/Calculator.test.js:16]: I have these tests in the operator test

**Review by GzimAsani — APPROVED** [review]
# STATUS: APPROVED :100:         
## Thank you for implementing all the project requirements, you've done a great job :trophy: There is nothing else to say other than... it's time to merge it :champagne:  :tada: Congratulations

Cheers and Happy coding! :clap:  :computer:  :wine_glass: 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**